src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -288,6 +292,10 @@ enum AccessibilityIdentifiers {
 	enum General {
 		static let exposureSubmissionNavigationControllerTitle = "ExposureSubmissionNavigationController"
 		static let image = "ExposureSubmissionIntroViewController.image"
+
+		static let primaryFooterButton = "General.primaryFooterButton"
+		static let secondaryFooterButton = "General.secondaryFooterButton"
+		static let cancelButton = "General.cancelButton"
```

Comment:
Are these used anywhere already or just set so far?

Reply:
Are live and kicking in `ENANavigationFooterView` as default for the footer buttons. I haven't touched the current custom settings of these in existing code to reduce noise in this PR.

The `ExposureSubmissionQRScannerViewController` uses the cancel button but not in any test yet. I planned something for the screenshots and postponed it until further working on screenshots.